src/pubkey.h

```diff
@@ -15,10 +16,12 @@
 
 /**
  * secp256k1:
- * const unsigned int PRIVATE_KEY_SIZE = 279;
- * const unsigned int PUBLIC_KEY_SIZE  = 65;
- * const unsigned int SIGNATURE_SIZE   = 72;
- *
+ */
+const unsigned int PUBLIC_KEY_SIZE             = 65;
```

Comment:
yikes, please don't put these in the global namespace in a very public header file.